.vitepress/theme/constants/team.ts

```diff
@@ -0,0 +1,108 @@
+export type TeamMember = CoreTeamMember | CommunityTeamMember;
+
+interface TeamMemberBase /* extends VitePress' `TeamMember` type, which is not exporeted */ {
+  id: string;
+  type: "core" | "community";
+  avatar: string;
+  name: string;
+  title?: string;
+  org?: string;
+  orgLink?: string;
+  desc?: string;
+  links?: {
+    icon:
+      | "discord"
+      | "facebook"
+      | "github"
+      | "instagram"
+      | "linkedin"
+      | "mastodon"
+      | "slack"
+      | "twitter"
+      | "x"
+      | "youtube"
+      | { svg: string };
+    link: string;
+    ariaLabel?: string;
+  }[];
+  sponsor?: string;
+}
+
+export interface CoreTeamMember extends TeamMemberBase {
+  type: "core";
+}
+
+export interface CommunityTeamMember extends TeamMemberBase {
+  type: "community";
+}
+
+export const TEAM_MEMBERS: TeamMember[] = [
```

Comment:
@Boshen 

Are there people who I should include in the list?

By the way, the words "core member" and "community member" is just tentative, so tell me if you have another thoughts (related to https://github.com/oxc-project/oxc-project.github.io/pull/16/files#r1398423787).